Sources/BlueTriangle/BlueTriangle.swift

```diff
@@ -40,6 +40,9 @@ final public class BlueTriangleConfiguration: NSObject {
         }
     }
 
+    /// Boolean value indicating whether user is a returning visitor.
+    @objc var isReturningVisitor: Bool = false
```

Comment:
Do you think `isNewUser: Bool = false` makes more sense insofar as I would expect that to better reflect the more typical situation?

Reply:
I think whatever makes to BT's terminology.